homeassistant/components/rest_command/__init__.py

```diff
@@ -58,6 +60,19 @@
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the REST command component."""
 
+    async def reload_service_handler(service: ServiceCall) -> None:
+        """Remove all rest_commands and load new ones from config."""
+        conf = await async_integration_yaml_config(hass, DOMAIN) or {}
```

Comment:
I mean something like this:
```suggestion
        conf = await async_integration_yaml_config(hass, DOMAIN)
        
        # conf will be None if the configuration can't be parsed
        if conf is None:
            return
```